src/ui/simulator/windows/message.cpp

```diff
@@ -72,7 +72,7 @@ class MessageProvider final : public Component::Spotlight::IProvider
 
     virtual void search(Spotlight::IItem::Vector& out,
                         const Spotlight::SearchToken::Vector& /*tokens*/,
-                        const Yuni::String& text = "") override
+                        const Yuni::String& /*tokens*/) override
```

Comment:
consider removing this parameter from function definition

Reply:
Tricky, so we restore the **text** parameter